app/src/cose.h

```diff
@@ -91,6 +196,36 @@ namespace scitt::cose
       }
       return false;
     }
+
+    // Returns a bool representing whether the input label is a known
+    // parameter in the context of a notary profile.
+    bool is_known_in_notary(
+      const std::variant<int64_t, std::string>& label) const
+    {
+      // List of known header parameters in a notary profile
+      auto notary_known_params(BASIC_HEADER_PARAMS);
```

Comment:
Is there a nicer way to combine lists?

Reply:
Not really,  but in this particular case you don't actually need the combined list, instead you could search in all three lists.

If you make them `std::std`s instead of vectors, this is even shorter as `BASIC_HEADER_PARAMS.contains(label) || EXTRA_HEADER_PARAMS.contains(label) || NOTARY_HEADER_PARAMS.contains(label)`